src/ansys/dpf/core/results.py

```diff
@@ -419,19 +419,19 @@ def split_by_body(self):
 
         Examples
         --------
-        >>> from ansys.dpf import core as dpf
-        >>> from ansys.dpf.core import examples
-        >>> model = dpf.Model(examples.download_all_kinds_of_complexity())
-        >>> disp = model.results.displacement
-        >>> fc_disp = disp.split_by_body.eval()
-        >>> len(fc_disp)
-        11
-        >>> fc_disp.get_mat_scoping().ids
-        <BLANKLINE>
-        ...1, 5, 6, 10, 2, 7, 8, 13, 4, 12, 15]...
-        >>> disp_mat_10 = fc_disp.get_field_by_mat_id(10)
-
-        """
+        # >>> from ansys.dpf import core as dpf
+        # >>> from ansys.dpf.core import examples
+        # >>> model = dpf.Model(examples.download_all_kinds_of_complexity())
+        # >>> disp = model.results.displacement
+        # >>> fc_disp = disp.split_by_body.eval()
+        # >>> len(fc_disp)
+        # 11
+        # >>> fc_disp.get_mat_scoping().ids
+        # <BLANKLINE>
+        # ...1, 5, 6, 10, 2, 7, 8, 13, 4, 12, 15]...
+        # >>> disp_mat_10 = fc_disp.get_field_by_mat_id(10)
+        #
+        # """
```

Comment:
I guess this needs to be uncommented to mark the comment termination no?

Reply:
Yes!